contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -1,15 +1,30 @@
 # This file was generated automatically by flowlogs-pipeline confgenerator
 log-level: error
+pipeline:
```

Comment:
So the output now puts the data structures back in natural order (instead of the golang default of lexicographical order); `pipeline` before `parameters`; `name` before `follows`; etc. Good!

Reply:
I think that's a nice side effect of #267 rather than this PR 😅